### PR TITLE
Adds sqlalchemy pool values for nova.

### PR DESCRIPTION
**Pull request by dguitarbite**
Add sqlalchemy pool values to avoid hardcoding DB pool values for nova.

b/p: https://github.com/crowbar/crowbar-openstack/pull/40

Conflicts:
    chef/data_bags/crowbar/bc-template-nova.json

### CODE REVIEW

**Comment by tboerger** [comment]
I have no idea if the default values are ok or usable but LGTM codewise. Maybe just drop the `Conflicts` stuff from the commit message.

**Comment by dguitarbite** [comment]
@tboerger Yes, dropping the conflicts stuff from the commit message. For the default values related discussion please check the crowbar-openstack PR.

**Comment by toabctl** [comment]
Let's wait with this PR until https://github.com/crowbar/crowbar-openstack/pull/40 is merged. And let's discuss it in the other PR.

File: chef/data_bags/crowbar/bc-template-nova.json [hunk 1]
```diff
@@ -67,7 +67,11 @@
       "db": {
         "password": "",
         "user": "nova",
-        "database": "nova"
+        "database": "nova",
+        "max_pool_size":,
```
Comment (aplanas) [inline chef/data_bags/crowbar/bc-template-nova.json]: this is not even a valid json : (

the same for the lines below

**Comment by toabctl** [comment]
Will be reopened when https://github.com/crowbar/crowbar-openstack/pull/126 is merged.

**Pull request closed without merging**